sources/payment-hub/target/classes/META-INF/spring-configuration-metadata.json

```diff
@@ -0,0 +1,131 @@
+{
+  "groups": [
+    {
```

Comment:
Shouldn't this under target/classes fall under .gitignore? If so, should this rather be inside a resources classpath from where it gets picked or is this json supposed to be generated by a build script?